package.json

```diff
@@ -21,9 +21,9 @@
   },
   "author": "Oussama Labrahmi <oussama@labrahmi.me>",
   "dependencies": {
-    "@clerk/nextjs": "^4.27.2",
-    "@hookform/resolvers": "^3.3.2",
-    "@prisma/client": "^5.3.1",
+    "@clerk/nextjs": "^4.29.3",
+    "@hookform/resolvers": "^3.3.4",
+    "@prisma/client": "^5.8.0",
     "@radix-ui/react-avatar": "^1.0.4",
     "@radix-ui/react-dialog": "^1.0.5",
     "@radix-ui/react-dropdown-menu": "^2.0.6",
```

Comment:
Please run `git checkout develop package.json` to reset package.json as it was... and do not change any of the dependencies versions unless it is strictly required for your task. 🙏🏻